src/providers/ecency/ecency.ts

```diff
@@ -237,10 +247,21 @@ export const addReport = async (type: 'content' | 'user', data: string) => {
   }
 };
 
-export const deleteAccount = async (username: string) => {
+
+/**
+* TOOD:
+* POST /private-api/request-delete
+* 
+* body: 
+* username:string
+* data:string
+* 
+**/
+export const deleteAccount = async (data: string, username: string) => {
```

Comment:
1. username should be first argument to avoid conflict from previous calls
2. use `ecencyApi` instead `api`